src/FirebaseArduino.cpp

```diff
@@ -17,12 +17,20 @@
 #include "FirebaseArduino.h"
 
 void FirebaseArduino::begin(const char* host, const char* auth) {
+  begin(String(host), String(auth));
```

Comment:
That is true in general. We are actually passing these through to Firebase.h which takes string objects though.

I think we can actually just drop the const char* methods. I think if we just take String& the compiler will use the String(const char*) constructor and make them for us.